README.md

```diff
@@ -30,3 +25,29 @@ yarn install
 ```
 yarn start
 ```
+
+## Run on Fork
```

Comment:
I had the fork setup working temporarily but then it broke with "internal RPC error". Tried wallet nonce data reset and whatnot but didnt wanna sink too much time